android/app/src/test/java/com/microsoft/codepush/common/utils/StringUtilsUnitTest.java

```diff
@@ -0,0 +1,24 @@
+package com.microsoft.codepush.common.utils;
+
+import org.junit.Test;
+
+import static com.microsoft.codepush.common.utils.StringUtils.isNullOrEmpty;
```

Comment:
There is default API for that. See [TextUtils.isEmpty](https://developer.android.com/reference/android/text/TextUtils.html#isEmpty(java.lang.CharSequence))

Reply:
@MatkovIvan Thanks for pointing on that!